### ISSUE TITLE
Add PowerShell Core Support

### ISSUE THREAD

**Issue by alagoutte** [AsBuiltReport/AsBuiltReport.VMware.vSphere#78]
When try to use AsBuiltReport on PowerShell Core, there is error about missing some VMware.PowerCLI cmdlets ( Get-VMHostHardware, Get-PatchBaseline,  Get-Compliance)

it will be better to "skip" (and don't have an fatal error)
Now PScribo is compatible with PowerShell Core

**System 
 - OS: Windows 10, Linux (Ubuntu), macOS
 - Windows PowerShell version 6.2.4

add diff of missing element if you "ignore" (See https://github.com/alagoutte/AsBuiltReport.VMware.vSphere/tree/Add-PowerShellCore-Support ) missing cmdlet

```95d94
< 1.4.1.2.3          Update Manager Compliance
127d125
329,330d324
< 1.9             VMware Update Manager
< 1.9.1            Baselines
715c709
< Number of NICs          : 2
---
> Number of NICs          : Unknown
729c723
< Uptime Days             : 30,11
---
> Uptime Days             : 30,03
797,805d790
< 1.4.1.2.3 Update Manager Compliance
< ------------------------------------------------------------------------------------------------------------------------
<
< Baseline                                Status
< --------                                ------
< Critical Host Patches (Predefined)     Unknown
< Non-Critical Host Patches (Predefined) Unknown
<
<
1100,1101c1085,1086
< Serial Number           : CZ3502LSAT
< Asset Tag               :
---
> Serial Number           : Unknown
> Asset Tag               : Unknown
1110c1095
< Number of NICs          : 2
---
> Number of NICs          : Unknown
1185,1193d1169
< 1.4.2.2.3 Update Manager Compliance
< ------------------------------------------------------------------------------------------------------------------------
<
< Baseline                                Status
< --------                                ------
< Critical Host Patches (Predefined)     Unknown
< Non-Critical Host Patches (Predefined) Unknown
<
<
3638,3653d3595
<
<
< 1.9 VMware Update Manager
< ------------------------------------------------------------------------------------------------------------------------
< The following sections detail the configuration of VMware Update Manager managed by vCenter Server bdx-sr-vcsa.demovsan.
< intra.
<
< 1.9.1 Baselines
< ------------------------------------------------------------------------------------------------------------------------
<
< Baseline                                                  Description
< --------                                                  -----------
< Critical Host Patches (Predefined)                        A predefined baseline for all critical patches for Hosts
< ESX Agent Manager :: 0cf991bd-bc58-4935-9419-34433eaf5e9e Baseline for agency: 0cf991bd-bc58-4935-9419-34433eaf5e9e and
<                                                           software: [334, 335, 336]owned by ESX Agent Manager
< Non-Critical Host Patches (Predefined)                    A predefined baseline for all non-critical patches for Hosts
```

**Comment by alagoutte** [comment]
VMware.PowerCLI 12.0 is available, it will may be help

> The VMware.VimAutomation.License module has been ported to work on Linux and macOS.

https://vdc-download.vmware.com/vmwb-repository/dcr-public/88aaead3-8435-4c36-877e-713c43b4ba28/df6ea25b-578a-4cd6-9917-437ab5e8294b/changelog.html

**Comment by tpcarman** [comment]
Unfortunately there is still no PS Core / 7 support for Get-VMHostHardware in PowerCLI 12.0

`Get-VMHostHardware: Cmdlet Get-VMHostHardware is not supported on PowerShell Core.`

**Comment by tpcarman** [comment]
Fixed in [v1.3.0](https://github.com/AsBuiltReport/AsBuiltReport.VMware.vSphere/commit/8cac3fd7f46f666d0493a47b740ac4012cd69385)
